services/api-server/requirements/_test.txt

```diff
@@ -323,6 +323,8 @@ wrapt==1.15.0
     # via aws-xray-sdk
 xmltodict==0.13.0
     # via moto
+aioresponses==0.7.4
```

Comment:
hmm this looks like you manually changed that.
we have a makefile there to do that automatically, which also sorts, etc...
```make touch reqs upgrade=aioresponses``` in this very folder.

In any case I can show you if it does not make sense

Reply:
I am not able to get this to work. Am I supposed to run this from the root folder or from within `/services/api-server`?

Reply:
This has been added according to our discussion offline